cmd/operator-ui/actions/blackducks.go

```diff
@@ -363,3 +369,22 @@ func (v BlackducksResource) Destroy(c buffalo.Context) error {
 	// Redirect to the blackducks index page
 	return c.Redirect(302, "/blackducks")
 }
+
+func validateBlackduck(blackduck *blackduckapi.Blackduck) error {
```

Comment:
@jdartigalongue can we do this validation on submit action in Javascript itself?